ngraph/core/src/op/scatter_nd_update.cpp

```diff
@@ -30,3 +33,79 @@ shared_ptr<Node> op::v3::ScatterNDUpdate::clone_with_new_inputs(const OutputVect
                                                 new_args.at(op::util::ScatterNDBase::INDICES),
                                                 new_args.at(op::util::ScatterNDBase::UPDATES));
 }
+
+namespace scatter
+{
+    template <element::Type_t ET>
+    bool evaluate(const HostTensorPtr& arg0,
```

Comment:
Why do we need to have `evaluate` methid for this operation?

Reply:
Because there is a model where this operation is in the constant sub-graph (ShapeOf sub-graph)